3_run_analysis.R

```diff
@@ -1,7 +1,7 @@
 # TODO:
 # Clean up sectors options
 
-port_col_types <- set_col_types(grouping_variables, "ddddccccddcl")
+# port_col_types <- set_col_types(grouping_variables, "ddddccccddcl")
```

Comment:
Is "comment in/out" the way to determine whether a run of this file is for the online/offline tool?

Reply:
@Clare2D can you explain a bit why we are commenting a line out instead of removing it? 

(I suspect it's because it helps run a different workflow and if so I would think of a way to parametrize this line.)

Reply:
I commented it out because it caused it to fail, and the test isn't very indicative of the issue. Could be rewritten as something more relevant